cli/cmd/alpha_clients.go

```diff
@@ -12,10 +12,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// alphaClientsOptions holds values for command line flags that apply to the alpha
```

Comment:
These changes are already merged in [4412](https://github.com/linkerd/linkerd2/pull/4412/files)